engine/src/io/data_parser/ParquetParser.cpp

```diff
@@ -108,9 +105,11 @@ std::unique_ptr<ral::frame::BlazingTable> parquet_parser::parse(
 		//columns_out[i].create_gdf_column(table_out.get_column(i));
 		// }
 		// }
-		return std::make_unique<ral::frame::BlazingTable>(std::move(result.tbl), result.metadata.column_names);
+		std::unique_ptr<ral::frame::BlazingTable> table_out = std::make_unique<ral::frame::BlazingTable>(std::move(result.tbl), result.metadata.column_names);
+		ral::frame::BlazingTableView table_out_view = table_out->toBlazingTableView();
+		return std::make_pair(std::move(table_out), table_out_view);
```

Comment:
return the table or return it as nullptr with view set not both

Reply:
see comments above